src/aap_eda/api/exceptions.py

```diff
@@ -70,7 +71,7 @@ class NoControllerToken(APIException):
 class TooManyControllerTokens(APIException):
     status_code = 422
```

Comment:
Should we use this `status.HTTP_422_UNPROCESSABLE_ENTITY` instead of just hard-coded integer? Or we could inherit from `Unprocessable` class on line 48 and customize detail field since they fall under the same error code. Thoughts?

Reply:
Agree